index.html

```diff
@@ -357,11 +357,11 @@ <h1>bam2.iobio</h1>
       });
 
       document.addEventListener('bed-file-selected', (event) => {
-        console.log(event.detail)
+        broker.bedText = event.detail.bedText;
```

Comment:
This needs to go in bam.iobio/index.html right? This index.html needs to be removed, as it can create confusion.

Reply:
I think so. I forgot why I kept the index.html in the `iobio-charts` repo.